scalahost/nsc/src/main/scala/scala/meta/internal/semantic/ConfigOps.scala

```diff
@@ -4,14 +4,18 @@ package semantic
 import scala.meta.internal.io.PathIO
 import scala.meta.internal.scalahost.ScalahostPlugin
 import scala.meta.io._
+import scala.meta.internal.semantic.schema.Message.Severity
```

Comment:
I opted to re-use the existing data structure since I'm lazy and writing enums in Scala is annoying and boilerplatey. I'll change.